bin/serverless.js

```diff
@@ -9,7 +9,6 @@ if (userNodeVersion >= 8) {
   const serverlessCli = require('@serverless/cli');
   if (serverlessCli.runningComponents()) {
     serverlessCli.runComponents();
-    return;
```

Comment:
This change would be heavily breaking for us. I believe that's not intended commit.

Reply:
will undo this as this was change after adding `.eslintrc.js`